mycroft/skills/skill_manager.py

```diff
@@ -86,7 +85,7 @@ def put(self, loader):
             LOG.info('Updating settings meta during runtime...')
         with self.lock:
             # Remove existing loader
-            self._queue == [e for e in self._queue if e != loader]
+            self._queue = [e for e in self._queue if e != loader]
```

Comment:
unrelated opportunistic fix, this was evaluating to True and my IDE flagged it

TODO: send upstream